eng/validation/build-arcadewithrepo.ps1

```diff
@@ -195,52 +210,105 @@ function Git-Command($repoName) {
     }
 }
 
-## If able to retrieve a build, get the SHA that it was built from
-$sha = Get-LastKnownGoodBuildSha -buildDefinitionId $buildDefinitionId
-
-## Clone the repo from git
-GitHub-Clone $targetRepoName
-
-## Create a branch from the repo with the given SHA. 
+## Global Variables
```

Comment:
These should be marked as $global;<var-name> to make it really clear they are global. it tripped me up in some functions as I didn't know where some of the used variables were coming from.

Reply:
I also use a bunch of the input parameters for the script from AzDO as global variables. Is there a better way of handling those, too? (Do the same thing with the same variable name?)

Reply:
Yeah, anything that is actually global should be marked as such, just makes it easier to know where things are coming from, and avoids errors of overwriting the global when you just meant to overwrite a local var.

Reply:
I global'd just about everything 😂